src/pages/ReviewPage.tsx

```diff
@@ -160,15 +163,18 @@ const CandidateContestResult = ({
         <Text
           key={candidate.id}
           aria-label={`${candidate.name}${
-            candidate.party ? `, ${candidate.party}` : ''
+            candidate.partyId
+              ? `, ${parties.find(p => p.id === candidate.partyId)!.name}`
+              : ''
           }${candidate.isWriteIn ? `, write-in` : ''}${
             array.length - 1 === index ? '.' : ','
           }`}
           wordBreak
           voteIcon
         >
           <strong>{candidate.name}</strong>{' '}
-          {candidate.party && `/ ${candidate.party}`}
+          {candidate.partyId &&
+            `/ ${parties.find(p => p.id === candidate.partyId)!.name}`}
```

Comment:
good call.

Reply:
What are your thoughts on abstracting this array find method vs initing a lookup object keyed by party id?

Reply:
I abstracted it out already. I think it could go either way, but I think we're getting lost in bikeshedding now. Let's move forward with the abstraction I just pushed?